src/Tribe/iCal.php

```diff
@@ -75,37 +75,58 @@ public function single_event_links() {
 		if ( is_single() && post_password_required() ) {
 			return;
 		}
-		echo '<div class="tribe-events-cal-links">';
-		echo '<a class="tribe-events-gcal tribe-events-button" href="' . Tribe__Events__Main::instance()->esc_gcal_url( tribe_get_gcal_link() ) . '" title="' . esc_attr__( 'Add to Google Calendar', 'the-events-calendar' ) . '">+ ' . esc_html__( 'Google Calendar', 'the-events-calendar' ) . '</a>';
-		echo '<a class="tribe-events-ical tribe-events-button" href="' . esc_url( tribe_get_single_ical_link() ) . '" title="' . esc_attr__( 'Download .ics file', 'the-events-calendar' ) . '" >+ ' . esc_html__( 'iCal Export', 'the-events-calendar' ) . '</a>';
-		echo '</div><!-- .tribe-events-cal-links -->';
+		$calendar_links = '<div class="tribe-events-cal-links">';
+		$calendar_links .= '<a class="tribe-events-gcal tribe-events-button" href="' . Tribe__Events__Main::instance()->esc_gcal_url( tribe_get_gcal_link() ) . '" title="' . esc_attr__( 'Add to Google Calendar', 'the-events-calendar' ) . '">+ ' . esc_html__( 'Google Calendar', 'the-events-calendar' ) . '</a>';
+		$calendar_links .= '<a class="tribe-events-ical tribe-events-button" href="' . esc_url( tribe_get_single_ical_link() ) . '" title="' . esc_attr__( 'Download .ics file', 'the-events-calendar' ) . '" >+ ' . esc_html__( 'iCal Export', 'the-events-calendar' ) . '</a>';
+		$calendar_links .= '</div><!-- .tribe-events-cal-links -->';
+
+		/**
+		 * Allow for complete customization of the iCal and gCal single-event links.
+		 *
+		 * @param string $calendar_links The HTML of the iCal and gCal single-event link buttons.
+		 */
+		echo apply_filters( 'tribe_events_single_links', $calendar_links );
```

Comment:
Not super stoked on the name of the hook - `tribe_events_single_links` is really generic and doesn't convey anything about it being the iCal/GCal link.